components/Downloads/DownloadHeader/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { FormattedMessage, injectIntl } from 'react-intl';
+import styles from './index.module.scss';
+import SectionTitle from '../../Common/SectionTitle';
+import type { WrappedComponentProps } from 'react-intl';
+
+type Props = {
+  // release?: NodeReleaseData;
```

Comment:
Mine has been merged, you can merge back latest `major/website-redesign` branch. 😄

Reply:
YEAH !!!